src/app/space/create/create-routing.module.ts

```diff
@@ -39,6 +39,17 @@ const routes: Routes = [
           featureName: 'Applications'
         }
       },
+      {
+        path: 'deployments',
```

Comment:
By having the menus display deployments conditionally and the routing always have deployments available, the page is can be visited in production by manually accessing the url [...]/create/deployments

Is there no way to drop the route conditionally as well?

Reply:
It's possible, but means there will either need to be some utility extracted out somewhere up the file hierarchy, or some duplicated logic to check for development/production. I asked @joshuawilson if it was sufficient to just visually hide it and leave the route in place and IIRC he was okay with that.

Reply:
Okay. If Joshua is fine then I have no objections either.

Reply:
I'm fine with having the routing in place and just hiding the menu option. The main thing is to not break production. A hidden route while under development sounds like a reasonable option.